.vscode/launch.json

```diff
@@ -147,7 +147,23 @@
 			"outFiles": [
 				"${workspaceFolder}/out/**/*.js"
 			]
-		}
+        },
+        {
+			"type": "chrome",
+			"request": "launch",
+            "name": "Run Extension Unit Tests",
+			"windows": {
+				"runtimeExecutable": "${workspaceFolder}/scripts/test-extensions-unit.bat"
+			},
+			"osx": {
+				"runtimeExecutable": "${workspaceFolder}/scripts/test-extensions-unit.sh"
+			},
+			"linux": {
+				"runtimeExecutable": "${workspaceFolder}/scripts/test-extensions.sh"
```

Comment:
why is this different for mac and linux? Usually they're the same

Edit: Below it looks like you renamed the file so I expect this just needs to be updated